team/andy-barrett.yaml

```diff
@@ -0,0 +1,20 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Andy Barrett
+  avatar: https://avatars.githubusercontent.com/andypbarrett
+  role: Research Associate
+  organizations:
+    - name: National Snow and Ice Data Center, University of Colorado
+  bio: >
+    A FEW SENTENCES ABOUT YOU, MULTIPLE LINES ARE FINE
```

Comment:
@andypbarrett don't forget this bit! once sentence is fine, or you can just leave it blank